lib/resources/LoginLinks.js

```diff
@@ -3,6 +3,7 @@
 const StripeResource = require('../StripeResource');
 
 module.exports = StripeResource.extend({
-  path: 'accounts/{accountId}/login_links',
+  path: 'accounts/{account}/login_links',
```

Comment:
this should be `accountId`, no?

Reply:
@rattrayalex-stripe Not as per OpenAPI, but we can override if necessary?